install_using_composable.sh

```diff
@@ -113,30 +113,32 @@ apiVersion: operators.coreos.com/v1alpha1
 kind: Subscription
 metadata:
   labels:
-    operators.coreos.com/composable-operator.turbonomic: ""
-  name: composable-operator
+    operators.coreos.com/t8c-certified.turbonomic: ""
+  name: t8c-certified
   namespace: ${NS}
 spec:
-  name: composable-operator
-  source: community-operators
+  name: t8c-certified
+  source: certified-operators
   sourceNamespace: openshift-marketplace
 EOF
-until oc get crd composables.ibmcloud.ibm.com >> /dev/null 2>&1; do sleep 5; done
+until oc get crd xls.charts.helm.k8s.io >> /dev/null 2>&1; do sleep 5; done
+
+sleep 30
```

Comment:
Actually if we do it in this order (where we wait for the XL CRD to exist before creating the composable subscription) that should solve the caching issue without the need for the extra sleep, so it should be safe to remove this line and keep the other changes.

Reply:
ok, I removed the sleep